contracts/Governance/VestingToken.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.5;
+
+/// Token vesting contract.
+/// Adapted from Melonport AG <team@melonport.com>
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+contract VestingToken {
+    using SafeMath for uint256;
+
+    ERC20 public token; // Radicle ERC20 contract
+    address public owner; // deployer; can interrupt vesting
```

Comment:
```suggestion
    ERC20 public immutable token; // Radicle ERC20 contract
    address public immutable owner; // deployer; can interrupt vesting
```
They never change, no need to keep them in storage

Reply:
Can you elaborate? Aren't they kept in storage regardless?

Reply:
No, the `immutable` variables are baked into the contract as constants when deployed

Reply:
Ah cool!